runtime-react/pom.xml

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.github.mcollovati</groupId>
+        <artifactId>quarkus-hilla-parent</artifactId>
+        <version>${revision}</version>
+    </parent>
+
+    <artifactId>quarkus-hilla-react</artifactId>
+    <name>Quarkus - Hilla React - Runtime</name>
+    <description>
+        <![CDATA[A Quarkus extension to run Hilla React applications on Quarkus.]]></description>
+    <dependencies>
+        <dependency>
+            <groupId>com.github.mcollovati</groupId>
+            <artifactId>quarkus-hilla</artifactId>
+            <version>${project.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.github.mcollovati</groupId>
+                    <artifactId>hilla-jandex</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Exclusion not needed, since factored out or?

Reply:
I don't understand the question.
Exclusion is required otherwise there will be two jandex jars

Reply:
Ok, but would be some dependency refactoring then be better?

Reply:
Could you elaborate the solution you have in mind?

Reply:
So that quarkus-hilla-react is not pulling in quarkus-hilla. So that either the deps are copied from quarkus-hilla, moved to the parent pom or creating a new pom between hilla and quarkus.

```mermaid
  graph TD;
      quarkus-hilla-parent-->quarkus-hilla;
      quarkus-hilla-parent-->quarkus-hilla-react;
```

```mermaid
  graph TD;
      quarkus-hilla-parent-->quarkus-hilla-runtime-parent;
      quarkus-hilla-runtime-parent-->quarkus-hilla;
      quarkus-hilla-runtime-parent-->quarkus-hilla-react;
`
``

Reply:
Ok, now I got it.
The second option looks better to me.
I would name it `quarkus-hilla-common` and put there basically all the extension code, so `quarkus-hilla` and `quarkus-hilla-react` will depend on it and bring in the correct jandex index.

Reply:
Yes, I was thinking to move the code